activestorage/lib/active_storage/analyzer/null_analyzer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
```

Comment:
There are no strings in this file.

Reply:
Rubocop doesn't make that distinction. And even if it could, of two bad options, I think I'd prefer to always add this when creating new files than add it when a later edit happens to introduce a string.